### ISSUE TITLE
Customizing Content Width in Learning Mode template does not work

### ISSUE THREAD

**Issue by dericleeyy** [Automattic/themes#8487]
### Steps to Reproduce
1. Edit the learning mode template
2. Click on Content Block in Main Content
3. Set a custom content width
4. Save

### What I Expected
The lessons to follow the custom content width set

### What Happened Instead
The styling is followed at first, but it reloads and resets the content width to fill up the entire screen

### PHP / WordPress / Sensei LMS version
WP 6.7.1
Sensei 4.24.4

### Screenshot / Video
![Image](https://github.com/user-attachments/assets/47d7bc3c-4c33-4b23-aaad-30881c5bbf88)
![Image](https://github.com/user-attachments/assets/6dc3e1a2-48b2-4b12-81e1-606384015d37)

### Context / Source

9121185-zen

**Comment by supernovia** [comment]
📌 REPRODUCTION RESULTS
- Tested on Atomic – Replicated

📌 FINDINGS/SCREENSHOTS/VIDEO
The site's main content width overrides anything set in the template
https://github.com/user-attachments/assets/0c719f4d-8f67-4e9e-aa28-140515faa5b6

📌 ACTIONS
- Triaged & prioritized

**Comment by dericleeyy** [comment]
@donnapep I don't think this is theme-dependent. I reproduced it on the Twenty Twenty-Four theme.

**Comment by donnapep** [comment]
Confirmed. Moving back to Sensei.
